src/app/locales/pt-br/hotkeys.yml

```diff
@@ -0,0 +1,12 @@
+# See the "Hotkeys" section of the "Translating" wiki page on how to properly translate hotkeys:
+# https://github.com/streamlink/streamlink-twitch-gui/wiki/Translating#28-hotkeys
+modifiers:
+    altKey: Alt
+    ctrlKey: Ctrl
+    metaKey: Meta
+    shiftKey: Shift
+codes:
+    ArrowLeft: Esquerda
+    ArrowRight: Direita
+    ArrowUp: Cima
+    ArrowDown: Baixo
```

Comment:
Forgot to comment on that...
If there are more hotkeys which need or should be translated, then you can add them here. See the wiki link above.

Reply:
I saw that. We use the same name as in US layout (ex: delete. page up, page down, home, end... they are all the same)